stylelint.config.js

```diff
@@ -0,0 +1,225 @@
+
+    'color-hex-case': ['lower', { severity: 'warning' }],
+    'color-hex-length': ['long', { severity: 'warning' }],
+
+    'font-family-name-quotes': ['always-unless-keyword', { severity: 'warning' }],
+
+    'font-weight-notation': ['numeric', { severity: 'warning' }],
+
+    'function-comma-newline-after': ['never-multi-line', { severity: 'warning' }],
+    'function-comma-newline-before': ['never-multi-line', { severity: 'warning' }],
+    'function-comma-space-after': ['always', { severity: 'warning' }],
+    'function-comma-space-before': ['never', { severity: 'warning' }],
+    'function-max-empty-lines': [0, { severity: 'warning' }],
+    'function-name-case': ['lower', { severity: 'warning' }],
+    'function-parentheses-newline-inside': ['never-multi-line', { severity: 'warning' }],
+    'function-parentheses-space-inside': ['never', { severity: 'warning' }],
+    'function-url-quotes': ['always', { severity: 'warning' }],
+    'function-whitespace-after': ['always', { severity: 'warning' }],
+
+    'number-leading-zero': ['always', { severity: 'warning' }],
+    'number-no-trailing-zeros': [true, { severity: 'warning' }],
+
+    'string-quotes': ['single', { severity: 'warning' }],
+
+    'length-zero-no-unit': [true, { severity: 'warning' }],
+
+    'unit-case': ['lower', { severity: 'warning' }],
+
+    'value-keyword-case': ['lower', { severity: 'warning' }],
+
+    'value-list-comma-space-after': ['always', { severity: 'warning' }],
+    'value-list-comma-space-before': ['never', { severity: 'warning' }],
+    'value-list-max-empty-lines': [0, { severity: 'warning' }],
+
+    'property-case': ['lower', { severity: 'warning' }],
+
+    'declaration-bang-space-after': ['never', { severity: 'warning' }],
+    'declaration-bang-space-before': ['always', { severity: 'warning' }],
+    'declaration-colon-newline-after': ['always-multi-line', { severity: 'warning' }],
+    'declaration-colon-space-after': ['always', { severity: 'warning' }],
+    'declaration-colon-space-before': ['never', { severity: 'warning' }],
+
+    'declaration-block-semicolon-newline-after': ['always-multi-line', { severity: 'warning' }],
+    'declaration-block-semicolon-space-after': ['always-single-line', { severity: 'warning' }],
+    'declaration-block-trailing-semicolon': ['always', { severity: 'warning' }],
+
+    'block-closing-brace-empty-line-before': ['never', { severity: 'warning' }],
+    'block-closing-brace-newline-after': ['always', { severity: 'warning' }],
+    'block-closing-brace-space-before': ['always-single-line', { severity: 'warning' }],
+    'block-opening-brace-newline-after': ['always-multi-line', { severity: 'warning' }],
+    'block-opening-brace-space-after': ['always-single-line', { severity: 'warning' }],
+    'block-opening-brace-space-before': ['always', { severity: 'warning' }],
+
+    'selector-attribute-brackets-space-inside': ['never', { severity: 'warning' }],
```

Comment:
also let’s add `selector-attribute-operator-space-after` and `selector-attribute-operator-space-before` with `"never"`